StoryMap/StoryMap/Common/Services/PhotoInputManager.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  PhotoInputManager.swift
+//  StoryMap
+//
+//  Created by Dory on 02/12/2021.
+//
+
+import UIKit
+import PhotosUI
+import Combine
+
+final class PhotoInputManager: NSObject {
+	public enum SourceType {
+		case camera
+		case photoLibrary
+	}
+	
+	// MARK: - Public properties
+	
+	var imageSubject = PassthroughSubject<UIImage, Never>()
+	
+	// MARK: - Public methods
+	
+	public func makeViewController(with sourceType: SourceType) -> UIViewController {
+		sourceType == .camera ? makePhotoCaptureController() : makeChooseImageController()
+	}
+	
+	// MARK: - Private methods
+	
+	private func makeChooseImageController() -> UIViewController {
+		var config = PHPickerConfiguration()
+		config.filter = .images
+		config.selectionLimit = 1
+		let picker = PHPickerViewController(configuration: config)
+		picker.delegate = self
+		return picker
+	}
+	
+	private func makePhotoCaptureController() -> UIViewController {
+		let imagePicker = UIImagePickerController()
```

Comment:
Any of these configs relevant?

- allowsEditing: Bool
- showsCameraControls: Bool